hil/ext/switches/dellnos9.py

```diff
@@ -179,7 +201,8 @@ def _get_port_info(self, interface):
         \r\n\r\n Native Vlan Id: 1512.\r\n\r\n\r\n\r\n
         MOC-Dell-S3048-ON#</command>\n</output>\n"
         """
-
+        if not self._is_port_on(interface):
+            self._port_on(interface)
```

Comment:
Leaving it on after getting the information doesn't seem correct.

Reply:
okay. I'll go without turning on the port at all. If the switchport is off that means there are no trunked or native vlans on it anyway.  (All VLANs must be removed from a switchport before it can be turned off)